### PR TITLE
[dscalarm] Add details in bridge handler error log to help find the problem

### PR DESCRIPTION
**Pull request by jimtng**
See https://community.openhab.org/t/dscalarm-spamming-log/155552

### CODE REVIEW

File: bundles/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/internal/handler/DSCAlarmBaseBridgeHandler.java [hunk 1]
```diff
@@ -380,7 +380,7 @@ public void checkThings() {
                 }
 
             } else {
-                logger.error("checkThings(): Thing handler not found!");
+                logger.error("checkThings(): Thing handler not found! Thing: {} {}", thing.getUID(), thing);
```
Comment (lolodomo) [inline bundles/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/internal/handler/DSCAlarmBaseBridgeHandler.java:383]: I have not checked what returns thing.toString() but what interesting additional info provides your second parameter ?

I see logger.error is massively used in that class while WARN level should have been used. Will make probably no sense to change only this one.

**Comment by lolodomo** [comment]
Rather than just enhancing the log message, can you try to not produce it when producing it is inappropriate ?
The user case is a thing that is disabled. If I am not wrong, such thing has no handler.

File: bundles/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/internal/handler/DSCAlarmBaseBridgeHandler.java (hunk 1 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/internal/handler/DSCAlarmBaseBridgeHandler.java:383]: As the cases do not have affect on openHAB as a whole, this should be changed to warn (some others might even to debug)

**Review by lsiepel — CHANGES_REQUESTED** [review]
Can you change the log levels?

**Comment by jimtng** [comment]
I haven't had a chance to look into this again earlier, as the changes @lolodomo asked would be a bigger change, but I can change the log level here and leave the more fundamental change for another PR, perhaps not by me. I am totally unfamiliar with this binding, and only added this extra log to assist someone on the forum.

**Comment by jimtng** [comment]
I've done s/logger.error/logger.warn/g

File: bundles/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/internal/handler/DSCAlarmBaseBridgeHandler.java (hunk 1 above)
Comment (jimtng) [reply bundles/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/internal/handler/DSCAlarmBaseBridgeHandler.java]: @lolodomo I've removed the additional argument thing.toString().

Comment (jimtng) [reply bundles/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/internal/handler/DSCAlarmBaseBridgeHandler.java]: also changed all logger.error to logger.warn

**Review by lsiepel — APPROVED** [review]
Much better. Still the logging can be improved even further, but I can understand that is outside the scope of this PR

**Pull request merged**